src/config/dynamic-theme-fixes.config

```diff
@@ -2898,32 +2898,83 @@ span[class^="burger"]
 codeforces.com
 
 INVERT
-#header > div:first-child img
-.action-link > div > img:first-child
-.roundbox-lt
-.roundbox-rt
-.roundbox-lb
-.roundbox-rb
+#userActivityGraph rect[data-items=""]
 .delete-resource-link
-a.contestParticipantCountLinkMargin > img
-.recent-actions > ul > li img[src*="hourglass"]
+.action-link img
+.non-decorated img[src$="tablesorter-bg.gif"]
 
 CSS
+#header > div:first-child > a::before {
+    content: "";
+    display: inline-block;
+    height: 65px;
+    width: 15px;
+    border-top-left-radius: 12px;
+    border-bottom-left-radius: 12px;
+    background-color: #fff;
+}
+#header > div:first-child > a img {
+    border-radius: 0 12px 12px 0;
+}
+#header > div:first-child img {
+    border-radius: 12px;
+}
+.datatable,
+.datatable > div,
+.roundbox {
+    border-radius: 6px;
+}
 .lt,
 .rt,
 .lb,
 .rb,
 .ilt,
-.irt {
+.irt,
+.roundbox-lt,
+.roundbox-rt,
+.roundbox-lb,
+.roundbox-rb {
     display: none;
 }
-input[type="submit"], input[type="button"], select {
-    border-style: solid;
+.backLava,
+.leftLava {
+    border-radius: 4.5px;
+}
+.backLava,
+.leftLava,
+.bottomLava,
+.cornerLava {
+    background: ${#BEBEBE} !important;
+}
+input[type="submit"],
+input[type="button"],
+select {
+    border-width: 1px;
+    border-style: inset;
+    border-radius: 3.5px;
+}
+input[type="submit"]:hover,
+input[type="button"]:hover {
+    border-style: outset;
+}
+input[type="submit"]:hover,
+input[type="button"]:hover {
+    transform: translate(1px, 1px);
+}
```

Comment:
This is a bunch of cosmetic improvements for all sorts of buttons, as codeforces.com has no styles for them and darkreader applies its own. This code makes all kinds of inputs look like in the light version.

Reply:
> as codeforces.com has no styles for them and darkreader applies its own.

We only apply styles for colors, we shouldn't change anything otherwise...

Reply:
Darkreader changes default inputs:
![изображение](https://user-images.githubusercontent.com/38539949/146064888-7eaa9b23-d294-4bbc-be7f-efaebae98d9b.png)
![изображение](https://user-images.githubusercontent.com/38539949/146064950-742322c8-dc1b-4b3c-9901-0c1159ad8e44.png)

Reply:
My version:
![изображение](https://user-images.githubusercontent.com/38539949/146065027-b0516598-ad29-4e42-b131-a7bbaf4fc07e.png)

Reply:
That's _intentional_, it's how browsers work(each platform/browser/OS has their own styling of these elements, but once you touch these elements, they will go to this ugly generic variant). We have a option in the UI to disable the styling of these elements.

Reply:
Ok, I see. I'll remove this part